taskt/Core/Automation/Commands/Folder Operation Commands/MoveCopyFolderCommand.cs

```diff
@@ -1,72 +1,72 @@
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Destination Folder Path")]
+        [InputSpecification("Enter or Select the destination folder path.")]
+        [SampleUsage(@"C:\temp\DestinationFolder || {ProjectPath}\DestinationFolder || {vDestinationFolderPath}")]
+        [Remarks("{ProjectPath} is the directory path of the current project.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFolderSelectionHelper)] 
         public string v_DestinationDirectory { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Create folder if destination does not exist")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Yes")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("No")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify whether the directory should be created if it does not already exist.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Yes** or **No**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Create Destination Folder (Option)")]
+        [PropertyUISelectionOption("Yes")]
+        [PropertyUISelectionOption("No")]
+        [InputSpecification("Specify whether the destination directory should be created if it does not already exist.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_CreateDirectory { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Delete folder if it already exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Yes")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("No")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify whether the folder should be deleted first if it is already found to exist.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Yes** or **No**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Delete Existing Folder (Option)")]
+        [PropertyUISelectionOption("Yes")]
+        [PropertyUISelectionOption("No")]
+        [InputSpecification("Specify whether the folder should be deleted first if it already exists in the destination directory.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_DeleteExisting { get; set; }
 
-        public MoveFolderCommand()
+        public MoveCopyFolderCommand()
         {
-            this.CommandName = "MoveFolderCommand";
-            this.SelectionName = "Move/Copy Folder";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "MoveCopyFolderCommand";
+            SelectionName = "Move/Copy Folder";
+            CommandEnabled = true;
+            CustomRendering = true;
```

Comment:
can you create default options for create directory and overwrite? I think they should both be set to "Yes"

Reply:
Please add these defaults in MoveCopyFileCommand too.

Reply:
done in both file and folder commands